datascience-notebook/Dockerfile

```diff
@@ -43,8 +57,13 @@ RUN conda config --add channels r && \
     'r-nycflights13=0.1*' \
     'r-caret=6.0*' \
     'r-rcurl=1.95*' \
+    'r-dbi=*' \
+    'r-scales=*' \
```

Comment:
We should really pin versions. We've had no end of pain with failed rebuilds when versions are allowed to float arbitrarily.